_layouts/base.html

```diff
@@ -6,13 +6,43 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+
+  <meta content="{{ site.title }}" property="og:site_name">
+  {% if page.title %}
+  <meta content="{{ page.title }}" property="og:title">
+  {% else %}
+  <meta content="{{ site.title }}" property="og:title">
+  {% endif %}
+  {% if post.image %}
+  <meta content="{{ site.url }}{{ site.url }}{{ post.image }}" property="og:image">
+  {% else %}
+  <meta content="{{ site.url }}/assets/images/color_white_debezium_type_600px.svg" property="og:image">
+  {% endif %}
+  <meta property="og:description" content="{{ site.description }}"> 
+  <meta property="og:url" content="{{ site.url }}{{ page.url }}"> 
+  <meta name="twitter:site" content="@{{ site.twitter_username }}">
+  <meta name="twitter:title" content="{{ page.title }}">
+  {% if page.author %}
+  <meta name="twitter:creator" content="@{{ page.author }}">
+  {% else %}
+  <meta name="twitter:creator" content="@debezium">
+  {% endif %}
+  {% if page.summary %}
+  <meta name="twitter:description" content="{{ page.summary }}">
+  {% else %}
+  <meta name="twitter:description" content="{{ site.description }}">
+  {% endif %}
+  <meta name="twitter:card"  content="summary_large_image">
```

Comment:
What is `summary_large_image`? It's not a variable, I suppose? Should it be removed?

Reply:
`summary_large_image` is for Twitter card view (https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image). Yes, It's not a variable of Jekyll. Twitter picks up large images from post as far as I am aware of.

Reply:
Oh, I see. Ok, let's give it a try then. Thanks!